core/arch/arm/plat-ls/main.c

```diff
@@ -215,7 +215,9 @@ void main_init_gic(void)
 	gic_init(gic_base + gicc_offset, gic_base + gicd_offset);
```

Comment:
This `gic_init() `should also be in same flag `#if !defined(CFG_WITH_ARM_TRUSTED_FW)` 
BTW, Did you faced any issue without this patch ?

Reply:
https://github.com/OP-TEE/optee_os/commit/0ee3f52ed191d21fb12ed5832d4b7929ac7001e4 removed it. We didn't see any issues without the patch because we don't use GIC before it is reinitialized by optee.